frontend/pages/[lng]/auth/client-details.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useEffect } from "react"
+import { useRouter } from "next/router"
+import axios from "axios"
+import styled from "@emotion/styled"
+import { getAccessToken } from "../../../lib/authentication"
+
+const Container = styled.section`
+  width: 100%;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 10px;
+  box-sizing: border-box;
+`
+
+const ClientButton = styled.div`
+  padding: 8px 16px;
+  background: #ffc107;
+  font-weight: bold;
+  font-size: 18px;
+  border-radius: 5px;
+  user-select: none;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.7;
+  }
+`
+
+const CancelButton = styled.div`
+  padding: 8px 16px;
+  background: #cccccc;
+  font-weight: bold;
+  font-size: 18px;
+  border-radius: 5px;
+  user-select: none;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.7;
+  }
+`
+
+const DeleteButton = styled.div`
+  padding: 8px 16px;
+  background: rgba(255, 0, 51, 0.5);
+  font-weight: bold;
+  font-size: 18px;
+  border-radius: 5px;
+  user-select: none;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.7;
+  }
+`
+
+const Main = styled.main`
+  padding: 16px;
+  box-sizing: border-box;
+  color: #146600;
+  border-radius: 5px;
+  margin-bottom: 16px;
+  overflow: auto;
+  background: #ffffff;
+
+  p {
+    margin: 0;
+    margin-bottom: 5px;
+    display: flex;
+
+    strong {
+      margin-right: 5px;
+    }
+  }
+`
+
+const Options = styled.section`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 16px;
+  background: #ffffff;
+  box-sizing: border-box;
+  padding: 8px;
+  border-radius: 5px;
+
+  div {
+    margin-left: 10px;
+  }
+
+  section {
+    display: flex;
+  }
+`
+
+const ConfirmOptions = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const ClientDetails = () => {
+  const router = useRouter()
+
+  const [client, setClient] = useState<any>({})
+  const [confirmDelete, setConfirmDelete] = useState(false)
+  const [confirmRegenerate, setConfirmRegenerate] = useState(false)
+
+  useEffect(() => {
+    showClient()
+  }, [])
+
+  const answerDelete = () => {
+    setConfirmDelete(false)
+    deleteClient()
+  }
+
+  const answerRegenerate = () => {
+    setConfirmRegenerate(false)
+    regenerateClient()
+  }
+
+  const showClient = async () => {
+    axios({
+      method: "GET",
+      url: `http://localhost:4000/auth/client?id=${router.query.id}`,
```

Comment:
Are these in `localhost:4000` in production as well? At least `apollo` uses `https://www.mooc.fi/api` in production, would this be `https://www.mooc.fi/auth` then?